README.md

```diff
@@ -62,6 +62,27 @@ with or without the `-e` option depending on whether you want to edit the *STRAU
 
 ***Note***: The `coqui` AI TTS is currently supported for python versions `<= 3.12` - If you are running `python==3.13`, you can downgrade to use this extra.
 
+## Citing *STRAUSS*
+
+Please cite *STRAUSS* using the JOSS [paper](https://joss.theoj.org/papers/10.21105/joss.07875):
+
+```
+@article{Trayford2025,
+  doi = {10.21105/joss.07875},
+  url = {https://doi.org/10.21105/joss.07875},
+  year = {2025},
+  publisher = {The Open Journal}, 
+  volume = {10},
+  number = {109},
+  pages = {7875},
+  author = {James W. Trayford and Samantha Youles and Chris Harrison and Rose Shepherd and Nicolas Bonne},
+  title = {strauss: Sonification Tools and Resources for Analysis Using Sound Synthesis},
+  journal = {Journal of Open Source Software}
+}
+```     
+
+If you have found *STRAUSS* useful, please star the repository on GitHub. 
```

Comment:
```suggestion
If you have found *STRAUSS* useful, please star the repository on [GitHub](https://github.com/james-trayford/strauss)! 

<a class="github-button" href="https://github.com/james-trayford/strauss" data-color-scheme="no-preference: light; light: light; dark: dark;" data-icon="octicon-star" data-size="large" aria-label="Star james-trayford/strauss on GitHub">Star</a>
```

I haven't checked button works yet! Will see how it looks in context if we merge into `cite-us`, can always roll back. link to this github (as I think this text get's replicated elsewhere, e.g. PyPI). Added exclamation mark for bonus whimsy.

Reply:
The button isn't working...